.github/workflows/sync-integration-templates.yaml

```diff
@@ -39,3 +39,4 @@ jobs:
               env:
                   AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
                   AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+                  AWS_DEFAULT_REGION: ${{ secrets.AWS_REGION }}
```

Comment:
is that part of this PR?

Reply:
It probably should be a part of another PR but tried to squeeze it in 😅